cegs_portal/search/urls.py

```diff
@@ -41,6 +41,11 @@
         views.v1.NonTargetRegEffectsView.as_view(),
         name="non_target_reo",
     ),
+    re_path(
```

Comment:
This URL isn't necessary anymore

Reply:
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/fc3cae97-b9ef-4f17-b729-5b9e98e34dd7)

Reply:
What do you have in mind that I point the url to?

Reply:
`search:non_target_reo`